lib/qa/authorities/linked_data/config/term_config.rb

```diff
@@ -111,27 +96,7 @@ def term_results_sameas_predicate
       def term_qa_replacement_patterns
         Config.config_value(term_config, :qa_replacement_patterns)
       end
-
-      # Are there replacement parameters configured for term fetch?
-      # @return [Boolean] true if there are replacement parameters configured for term fetch; otherwise, false
-      def term_replacements?
-        term_replacement_count.positive?
-      end
-
-      # Return the number of possible replacement values to make in the term URL
-      # @return [Integer] the configured number of possible replacements in the term url
-      def term_replacement_count
-        term_replacements.size
-      end
-
-      # Return the replacement configurations
-      # @return [Hash] the configurations for term url replacements
-      def term_replacements
-        return @term_replacements unless @term_replacements.nil?
-        @term_replacements = {}
-        @term_replacements = term_url_mappings.select { |k, _v| !term_qa_replacement_patterns.value?(k.to_s) } unless term_config.nil? || term_url_mappings.nil?
-        @term_replacements
-      end
+      alias qa_replacement_patterns term_qa_replacement_patterns
```

Comment:
#term_replacements?, #term_replacement_count, #term_replacements, and #term_subauthority_replacement_pattern are no longer used.  The processing of replacements happens in AuthorityUrlService and IriTemplate::UrlConfig, and IriTemplate::VariableMap.  These methods and there tests have been removed.